tests/functional/src/test/java/org/mule/functional/classloading/isolation/maven/MavenArtifactMatcherPredicateTest.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.functional.classloading.isolation.maven;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import org.mule.tck.size.SmallTest;
+
+import java.util.function.Predicate;
+
+import org.junit.Test;
+
+/**
+ * Tests the {@link java.util.function.Predicate} for {@link MavenArtifact}
+ */
+@SmallTest
+public class MavenArtifactMatcherPredicateTest
```

Comment:
Check that some comments form previous class applies to this and other test classes